File: Prácticas/Practica_23/Py Practica_23/Archivo CSV.py
import csv


class ArchivoCSV:

    def escribir(self):
        with open("datos.csv", mode="w", newline="") as archivo:
            writer = csv.writer(archivo)
            writer.writerow(["Nombre", "Edad"])
            writer.writerow(["Ana", 22])
            writer.writerow(["Luis", 18])

    def leer(self):
        print("Contenido del CSV:")
        with open("datos.csv", mode="r") as archivo:
            reader = csv.reader(archivo)
            for fila in reader:
                print(fila)


# MAIN
if __name__ == "__main__":
    a = ArchivoCSV()
    a.escribir()
    a.leer()

File: Prácticas/Practica_23/C++ Practica_23/src/ArchivoCSV.cpp
#include "ArchivoCSV.h"
#include <fstream>
#include <iostream>

void ArchivoCSV::escribir()
{
    ofstream archivo("datos.csv");

    archivo << "Nombre,Edad\n";
    archivo << "Ana,22\n";
    archivo << "Luis,18\n";

    archivo.close();
}

void ArchivoCSV::leer()
{
    ifstream archivo("datos.csv");
    string linea;

    while(getline(archivo,linea))
        cout << linea << endl;

    archivo.close();
}